wiki/Spectate/en.md

```diff
@@ -0,0 +1,9 @@
+---
+stub: true
+---
+
+# Spectate
+
+Spectate, or spec, is when someone watches another player playing a [beatmap](/wiki/Beatmaps) in real time (with approx. 500ms delay). To spectate, open the extended chat and click on a grey user panel.
+
+For this to work, you will need to have a copy of the exact same beatmap the player you wish to spectate is playing. When someone you are spectating starts playing a new beatmap, it will be announced in a private `#spectator` channel. However, if you have an [osu!supporter](/wiki/osu!supporter) tag, the beatmapset will be downloaded and processed for you.
```

Comment:
```suggestion
For this to work, you will need to have a copy of the exact same beatmap the player you wish to spectate is playing. When someone you are spectating starts playing a new beatmap, it will be announced in a private `#spectator` channel. However, if you have an [osu!supporter](/wiki/osu!supporter) tag, the beatmapset will be automatically downloaded via osu!direct and processed for you.
```
maybe worth mentioning osu!direct here